my-app/src/app/courses-page/course-item/course-item.component.css

```diff
@@ -187,4 +187,115 @@
     .delete-button {
         margin-left: 0;
     }
-}
\ No newline at end of file
+}
+
+::ng-deep .cdk-overlay-connected-position-bounding-box{
+    position: absolute;
+    background-color: rgb(203, 208, 212);
+}
+
+::ng-deep .cdk-overlay-pane {
```

Comment:
avoid ng-deep. Move those styles to the global file. You can name it `_overlay.scss`. Also do not override cdk styles. Apply your own class